chaincfg/init_test.go

```diff
@@ -121,21 +121,21 @@ var (
 				Id:          "Abstain",
 				Description: "Abstain voting for Pedro",
 				Bits:        0x0,   // 0b0000
-				IsIgnore:    false, // XXX this is the invalid bit
+				IsAbstain:   false, // XXX this is the invalid bit
```

Comment:
Now i'm really confused, I can understand that some invalid bit pattern would be an error and be interpreted as abstaining, but why is bits=0 an error?